Related works.md

```diff
@@ -15,3 +15,8 @@
 - General semantics (Korzybski)
 - Tektology (Bogdanov)
 - Synergetics (Fuller, Haken)
+- Wholeness and Implicate Order (Bohm)
+https://en.wikipedia.org/wiki/Wholeness_and_the_Implicate_Order
+
+Category Theory
```

Comment:
I can see this fitting, as a "language for connecting diverse fields".  I'd prefer to link to a resource like Wikipedia though.  This can maybe fit up by general system theory.